Clarify what this is exactly

nickavem opened the pull request:
The only reason I even knew this existed was GNOME software, and even then I didn't know exactly what it was, guessing it was some sort of other GE-like version. Exact wording and translations should be changed, but I think this specificity will make things much clearer.

Signed-off-by: Andrew Thurman <ajtbecool@gmail.com>

nickavem committed:
Clarify what this is exactly

Signed-off-by: Andrew Thurman <ajtbecool@gmail.com>

gasinvein commented on com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml:
```diff
@@ -5,9 +5,9 @@
-  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
-  <summary xml:lang="pt_BR">Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</summary>
-  <summary xml:lang="es">Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</summary>
+  <summary>Daily builds of Proton built on the Flatpak runtime</summary>
```
It's not daily, it's built from current latest tag.

nickavem committed:
Update com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml

Whoops! I knew it was the beta builds but wrote latest instead. My English failing me again.

gasinvein commented on com.valvesoftware.Steam.CompatibilityTool.Proton.metainfo.xml:
```diff
@@ -5,9 +5,9 @@
-  <summary>Compatibility tool for Steam Play based on Wine and additional components</summary>
-  <summary xml:lang="pt_BR">Ferramenta de compatibilidade para Steam Play baseada no Wine e componentes adicionais</summary>
-  <summary xml:lang="es">Herramienta de compatibilidad para Steam Play basada en Wine y componentes adicionales</summary>
+  <summary>Latest upstream builds of Proton built on the Flatpak runtime</summary>
```
@nanonyme What do you think? How we should name it so it would be clear what it is? "Latest Proton build on top of Freedesktop SDK" maybe?

nanonyme replied:
Can we have both freedesktop-sdk and flatpak mentioned or is it too long?

gasinvein replied:
"Latest Proton build on top of Freedesktop SDK flatpak runtime", then?

gasinvein commented:
@KingAnde Also please switch base branch to `beta` and rebase. I'd like all changes to go through `beta` branch, similar to what Steam flatpak does.
